README.md

```diff
@@ -61,6 +61,75 @@ Table of Contents
 
 ## Things to be aware of when writing code
 
+###Writing secure software
```

Comment:
You need to add a space here "###Writing secure software" between the ### and the text. Nice section, but I would not put it in the beginning of the ## Things to be aware of when writing code, maybe after Testing?

Reply:
I believe it is important to have it early on in the guidelines. A common problem related to security is that it is not thought of early in the process, but really it should be taken into consideration in the very beginning. Having it early in the guidelines will hopefully make more people think of it. To me it makes sense to have it under "Things to be aware of when writing code", but I'm also up for moving it to a separate section